tests/utils_test.py

```diff
@@ -10,7 +10,7 @@ def setUp(self):
         """Set up the test case with example music files"""
         # Example music files
```

Comment:
To get the CI working again I think you should add "music21" to pyproject.toml in the `test` dependencies section.

Reply:
music21 dependency will be there in my next push